sonarts-core/src/runner/processRequest.ts

```diff
@@ -37,6 +38,21 @@ export function processRequest(inputString: string) {
   const input = JSON.parse(inputString);
   let program = createProgram(input.tsconfig);
 
+  const filesNumber: number = input.filepaths.length;
+  let currentFile: string;
+  let processedFilesCounter = 0;
+
+  const logProgress = () => {
+    console.warn(
```

Comment:
As discussed, for the time being we are using stderr (despite being an hack) because we plan to refactor communication between Java and Node.js. I will add a comment to explain this.